pkg/agent/flowexporter/types.go

```diff
@@ -36,3 +41,11 @@ type Connection struct {
 	DestinationPodNamespace string
```

Comment:
Add some comments somewhere indicating Pod name/NS is filled only when the Pod is local?

Reply:
> A general question - have we observed higher CPU usage/contention when flow exporting is enabled?

I did not look closely into CPU utilization stats yet. I will compare the following scenarios w.r.t cpu utilization and percentage of used and runnable times for processes:
- No flow exporter
- Flow exporter with no workload
- Flow exporter with workload.